src/electron/views/screenshot/screenshot-view.tsx

```diff
@@ -1,13 +1,31 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { NamedFC } from 'common/react/named-fc';
+import { ScreenshotData } from 'common/types/store-data/unified-data-interface';
+import { isEmpty, isUndefined } from 'lodash';
 import * as React from 'react';
 
-export type ScreenshotViewProps = {};
-export const ScreenshotView = NamedFC<ScreenshotViewProps>('ScreenshotView', props => {
-    return (
-        <div>
-            <h1 tabIndex={0}>This is the screenshot view</h1>
-        </div>
-    );
+import { Screenshot } from './screenshot';
+
+export type ScreenshotViewProps = {
+    screenshotData: ScreenshotData;
+};
+
+const screenshotAltText: string = 'axe-android results screenshot with highlighted components';
+
+export const ScreenshotView = NamedFC<ScreenshotViewProps>('ScreenshotView', (props: ScreenshotViewProps) => {
+    if (isUndefined(props.screenshotData) || isEmpty(props.screenshotData)) {
+        return (
+            <div>
+                <p>Screenshot for scan is unavailable</p>
+            </div>
+        );
+    } else {
+        return (
+            <div>
+                <h1 tabIndex={0}>Page state screenshot</h1>
```

Comment:
2 questions here:
- is it **Page state screenshot** the right string here? (I didn't find figma mocks for this feature)
- is the `tabIndex` on purpose?

Reply:
1. tabIndex is on purpose; the screen reader I tested with skipped the heading if it wasn't there on my last PR
2. I got this phrase from the figma

Reply:
There are some figma mocks here: https://www.figma.com/file/Bugs8uNCHdCzOKzhq06nNK2o/Mobile---User-flow?node-id=249%3A2058

Reply:
- The tabIndex sounds odd, can we double check? (I can help with this from my box)
- Can we confirm? It sounds odd to me

Reply:
I checked with NVDA and I'm able to reach the header with arrow keys and the headings shortcut (I removed the `tabIndex` value locally to test)